components/common/hooks/useOpenCloudPurchaseModal.ts

```diff
@@ -29,6 +29,7 @@ export default function useOpenCloudPurchaseModal(options: OpenPurchaseModalOpti
             dialogType: PurchaseModal,
             dialogProps: {
                 callerCTA: telemetryProps.trackingLocation,
+                isInitialPlanMonthly: (isMonthlyPlan === undefined) ? true : isMonthlyPlan,
```

Comment:
I initially tried `isInitialPlanMonthly: isMonthlyPlan || true`,  but it doesn't work because if `isMonthlyPlan = false` (the user selected "yearly" on the pricing modal before pressing the update button to arrive at the purchase modal), then it would still default to true (false || true).  However `isMonthlyPlan ?? true` works so I'll change it to that. Thanks